roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -37,12 +39,12 @@ vcpu_pin_set = "1-31"
 {% endif -%}
 
 # Services offered #
-s3_host={{ endpoints.keystone }}
-ec2_host={{ endpoints.keystone }}
-ec2_dmz_host={{ endpoints.keystone }}
-ec2_url=http://{{ endpoints.nova }}:8773/services/Cloud
-cc_host={{ endpoints.keystone }}
-nova_url=http://{{ endpoints.nova }}:8774/v1.1/
+s3_host={{ endpoints.main }}
+ec2_host={{ endpoints.main }}
+ec2_dmz_host={{ endpoints.main }}
+ec2_url=http://{{ endpoints.main }}:8773/services/Cloud
```

Comment:
This port should probably be a variable as well, although I'm not sure we're actually offering ec2 API capability any more.